include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -17,32 +17,35 @@
 
 #include <cstdint>
 
+#include "internal/esimd_radix_sort_utils.h"
 #include "internal/esimd_radix_sort_dispatchers.h"
 
 namespace oneapi::dpl::experimental::kt::esimd
 {
 
-template <bool __is_ascending = true, ::std::uint8_t __radix_bits = 8, typename _KernelParam, typename _Range>
+template <bool __is_ascending = true, ::std::uint8_t __radix_bits = 8, typename _KernelParam, typename _KeysRng>
 sycl::event
-radix_sort(sycl::queue __q, _Range&& __rng, _KernelParam __param = {})
+radix_sort(sycl::queue __q, _KeysRng&& __keys_rng, _KernelParam __param = {})
 {
-    if (__rng.size() < 2)
+    if (__keys_rng.size() < 2)
         return {};
 
-    return __impl::__radix_sort<__is_ascending, __radix_bits>(__q, ::std::forward<_Range>(__rng), __param);
+    auto __pack = __impl::__utils::__rng_pack{oneapi::dpl::__ranges::views::all(::std::forward<_KeysRng>(__keys_rng))};
+    return __impl::__radix_sort<__is_ascending, __radix_bits>(__q, ::std::move(__pack), __param);
 }
 
-template <bool __is_ascending = true, ::std::uint8_t __radix_bits = 8, typename _KernelParam, typename _Iterator>
+template <bool __is_ascending = true, ::std::uint8_t __radix_bits = 8, typename _KernelParam, typename _KeysIterator>
```

Comment:
Somewhere in oneDPL code we use type name `ItKeys`
Looks like it's shorter variant.

Reply:
There are many options: `_Iterator`, `_Iter`, `_It` etc. This layer mimics `glue_*_impl.h` headers, e.g.: https://github.com/oneapi-src/oneDPL/blob/d23421606eb40a41d6554c7094eac44fa83ac43d/include/oneapi/dpl/pstl/glue_algorithm_impl.h#L732
So, it can also changed to `_RandomAccessIterator`. I would rather do this cosmetic change later, in consequent PRs. Currently the names denote that the Iterators/Ranges belong to keys or values.